src/admin-views/settings/tickets-commerce/gateways/container.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Template to display a list of featured gateways.
+ *
+ * @since TBD
+ *
+ * @var Tribe__Tickets__Admin__Views $this              Template object.
```

Comment:
Here it's not an instance of `Tribe__Tickets__Admin__Views` it's an instance of `Tribe__Template`